docs/guides/software-lifecycle/security/container-security/README.mdx

```diff
@@ -0,0 +1,130 @@
+import CodeBlock from '@theme/CodeBlock';
+import PreCommitConfigSource from '!!raw-loader!./.pre-commit-config.yml';
+import DependabotSource from '!!raw-loader!./dependabot.yml';
+
+# Container Security
+
+<pre align="center">Comprehensive guide to scanning container images for security vulnerabilities using pre-commit hooks and automated repository scanning tools.</pre>
```

Comment:
Hi @riverma explicit mention of **Docker** in the title is probably necessary. Reasoning; there are [many (OCI-compliant) alternatives to Docker](https://www.cloudzero.com/blog/docker-alternatives/) which could also be scanned for security vulnerabilities.

Reply:
The [Open Container Initiative (OCI)](https://opencontainers.org/) is a standard started and promoted by Docker, amongst others. Many on Lab are now using [Podman](https://docs.podman.io/en/latest/), which reports to be OCI-compliant, in place of Docker. In fact, I believe it is mandated going forward on AWS for some teams. [Here is more information on Podman origins.](https://www.redhat.com/en/topics/containers/what-is-podman) 

Thinking off-the-grid, what about making it an OCI-compliant container security guide?

Reply:
Hi @lewismc @jpl-jengelke - thought about your suggestions, discussed a bit with @lylebarner and ended up swapping to the [grype toolkit](https://github.com/anchore/grype), which is OGC compliant and not Docker specific. Moreover, it can scan non-containers as well.